encoding/thrift/thriftrw-plugin-yarpc/fx.go

```diff
@@ -33,9 +33,24 @@ const fxTemplate = `
 <$pkgname := printf "%sfx" (lower .Name)>
 package <$pkgname>
 
-<$yarpc  := import "go.uber.org/yarpc">
+<$transport := import "go.uber.org/yarpc/api/transport">
 <$thrift := import "go.uber.org/yarpc/encoding/thrift">
 <$client := import .ClientPackagePath>
+<$fx := import "go.uber.org/fx">
+
+// Params defines the dependencies for the <.Name> client.
+type Params struct {
+	<$fx>.In
+
+	Provider <$transport>.ClientConfigProvider
```

Comment:
I don't want to expose the transport package to end-users: can we alias or somehow move this type up to `yarpc.ClientConfig`?